README.md

```diff
@@ -54,7 +54,8 @@ Docker image, the file should look something like the following:
           "dev-mode",
           "clang-tidy",
           "cppcheck",
-          "ci-unix"
+          "ci-unix",
+          "include-what-you-use"
```

Comment:
Why is it only added to the user presets but not the one used in CI?

Reply:
Fixed by https://github.com/SpaceTeam/STS1_COBC_SW/pull/253/commits/f3240eef5d1841a70cb28167aa667d8965538130